tests/local/network.go

```diff
@@ -91,81 +85,66 @@ func NewLocalNetwork(warpGenesisFile string) *LocalNetwork {
 	_, err = os.Stat(warpGenesisFile)
 	Expect(err).Should(BeNil())
 
-	anrConfig := runner.NewDefaultANRConfig()
-	anrConfig.GlobalCChainConfig = warpEnabledChainConfig
-	manager := runner.NewNetworkManager(anrConfig)
+	subnetA := subnetEvmTestUtils.NewTmpnetSubnet(
+		"A",
+		warpGenesisFile,
+		subnetEvmTestUtils.DefaultChainConfig,
+		subnetANodes...,
+	)
 
-	// Construct the network using the avalanche-network-runner
-	_, err = manager.StartDefaultNetwork(ctx)
-	Expect(err).Should(BeNil())
-	err = manager.SetupNetwork(
-		ctx,
-		anrConfig.AvalancheGoExecPath,
-		[]*rpcpb.BlockchainSpec{
-			{
-				VmName:      evm.IDStr,
-				Genesis:     warpGenesisFile,
-				ChainConfig: warpChainConfigPath,
-				SubnetSpec: &rpcpb.SubnetSpec{
-					SubnetConfig: "",
-					Participants: subnetANodeNames,
-				},
-			},
-			{
-				VmName:      evm.IDStr,
-				Genesis:     warpGenesisFile,
-				ChainConfig: warpChainConfigPath,
-				SubnetSpec: &rpcpb.SubnetSpec{
-					SubnetConfig: "",
-					Participants: subnetBNodeNames,
-				},
-			},
-		},
+	subnetB := subnetEvmTestUtils.NewTmpnetSubnet(
+		"B",
+		warpGenesisFile,
+		subnetEvmTestUtils.DefaultChainConfig,
+		subnetBNodes...,
 	)
-	Expect(err).Should(BeNil())
 
-	// Issue transactions to activate the proposerVM fork on the chains
-	globalFundedKey, err := crypto.HexToECDSA(fundedKeyStr)
+	network := subnetEvmTestUtils.NewTmpnetNetwork(
+		"teleporter-test-local-network",
+		append(subnetANodes, append(subnetBNodes, extraNodes...)...),
+		tmpnet.FlagsMap{},
+		subnetA,
+		subnetB,
+	)
+	Expect(network).ShouldNot(BeNil())
+
+	avalancheGoBuildPath, ok := os.LookupEnv("AVALANCHEGO_BUILD_PATH")
+	Expect(ok).Should(Equal(true))
+
+	ctxWithTimeout, cancelBootstrap := context.WithTimeout(ctx, time.Minute*5)
+	err = tmpnet.BootstrapNewNetwork(
+		ctxWithTimeout,
+		os.Stdout,
+		network,
+		"",
+		avalancheGoBuildPath+"/avalanchego",
+		avalancheGoBuildPath+"/plugins",
+	)
+	defer cancelBootstrap()
 	Expect(err).Should(BeNil())
-	setupProposerVM(ctx, globalFundedKey, manager, 0)
-	setupProposerVM(ctx, globalFundedKey, manager, 1)
 
-	// Create the ANR client
-	logLevel, err := logging.ToLevel("info")
+	fundedKeyBytes, err := hex.DecodeString(fundedKeyStr)
 	Expect(err).Should(BeNil())
-
-	logFactory := logging.NewFactory(logging.Config{
-		DisplayLevel: logLevel,
-		LogLevel:     logLevel,
-	})
-	zapLog, err := logFactory.Make("main")
+	fundedKeySecp256k1, err := secp256k1.ToPrivateKey(fundedKeyBytes)
 	Expect(err).Should(BeNil())
+	network.PreFundedKeys = append(network.PreFundedKeys, fundedKeySecp256k1)
```

Comment:
Does `network.PreFundedKeys` have a default key already set? If so, we can likely get rid of the `fundedKeyStr` set up here all together, and if not, I'd recommend making a new slice here so that it's clear it's expected to start empty.

Reply:
`network.PreFundedKeys` seems to have a bunch of keys, but I tried to use them instead of the `fundedKeyStr` one and it didn't seem to have enough balance to run through all the tests.

I realized that there's nothing to be gained by appending the `fundedKeyStr` one to the `network.PrefundedKeys`, so I cut that out in https://github.com/ava-labs/teleporter/pull/401/commits/f5a74112e90ec7fe43ff663511d9f5cb8369e339.

It's possible we might be able to use the `PreFundedKeys` to run the tests, but for now I just left it working with `fundedKeyStr` to opt for less work.